packages/blockchain-utils-validation/src/blockchains/__tests__/ethereum.test.ts

```diff
@@ -54,21 +54,22 @@ const send = (provider: any, data: any): Promise<any> =>
   )
 const provider: any = ganache.provider(GANACHE_CONF)
 
-const lazyProvider = () => provider // Required for the Jest mock below
-jest.mock('@ethersproject/providers', () => {
-  const originalModule = jest.requireActual('@ethersproject/providers')
-  const getNetwork = (): any => {
-    return {
-      _defaultProvider: (): any => {
-        return new originalModule.Web3Provider(lazyProvider())
-      },
-    }
-  }
-  return {
-    ...originalModule,
-    getNetwork,
-  }
-})
+// TODO ESM Mocking
```

Comment:
can we put a ticket # in the TODO here?  Can be one ticket for tracking all the little odds and ends and todos that are left to get this ready to go.  That way when we can grep for `TODO(<ticket number>)` and see nothing show up we know we've gotten everything

Reply:
ditto for all other TODOs in this file (and elsewhere in this PR)